generate_schedule.py

```diff
@@ -199,8 +199,10 @@ def generate_schedule(args):
 
     ts_per_hour = datetime.timedelta(hours=1) / s.interval
 
+    assert len(s.constants.grid_connectors) == 1, "Only one grid connector supported"
+
     # compute flexibility potential (min/max) for each timestep
-    flex = generate_flex_band(s, args.core_standing_time)
+    flex = generate_flex_band(s, gc_index=0, core_standing_time=args.core_standing_time)
```

Comment:
use gcID instead: `gcID = list(s.world_state.grid_connectors.keys())[0]`
Please define the gcID separately and not in the function call. Looks cleaner to me. Maybe adjust the comment above: `compute flexibility potential (min/max) of single grid connector for each timestep`